src/plugins/dashboard/public/application/_dashboard_app.scss

```diff
@@ -54,6 +54,11 @@
   margin-left: $euiSizeL + $euiSizeXS;
 }
 
+.dshCopyToParagraph {
+  max-width: 400px;
```

Comment:
I still need to run this locally, but I've led you astray here. There is a `maxWidth` prop on `EuiModal` that can be used for this case.

Reply:
Ah, it turns out this setting was actually missing from the core service I was using to show the modal. I've added the option and it works correctly